test/tech/v3/libs/smile_ml_test.clj

```diff
@@ -34,11 +34,11 @@
 
 ;; take  long
 
-;; (deftest smile-regression-autogridsearch-test
-;;   (doseq [regression-model smile-regression-models]
-;;     (verify/auto-gridsearch-regression {:model-type regression-model 0.5 })))
+ ;; (deftest smile-regression-autogridsearch-test
+ ;;   (doseq [regression-model smile-regression-models]
+ ;;     (verify/auto-gridsearch-regression {:model-type regression-model} 0.5)))
 
 
-;; (deftest smile-classification-autogridsearch-test
-;;   (doseq [classify-model smile-classification-models]
-;;     (verify/auto-gridsearch-classification {:model-type classify-model} 0.5)))
+ ;; (deftest smile-classification-autogridsearch-test
+ ;;   (doseq [classify-model smile-classification-models]
+ ;;     (verify/auto-gridsearch-classification {:model-type classify-model} 0.5)))
```

Comment:
Are tests disabled?

Reply:
These were always disabled. But they pass, you can enable them.

Reply:
OK, I will enable them again after this lands; it was my mistake to have them disabled.  At the very least a nice sanity check.

Reply:
I did so in my last commit.